src/messages/direct.rs

```diff
@@ -25,7 +25,6 @@ use std::{
 };
 
 /// Direct message content.
-#[cfg_attr(feature = "mock_serialise", derive(Clone))]
 #[derive(Eq, PartialEq, Serialize, Deserialize)]
```

Comment:
need to remove that line since it is not necessary if mock_serialisation is off or gone.
Same for others

Reply:
No i misread.